tests/conftest.py

```diff
@@ -358,3 +360,24 @@ def resource_quota(api_instance, quota={}, namespace=TENANT_NAME):
     body = client.V1ResourceQuota(spec=resource_quota_spec, metadata=name_object)
     api_instance.create_namespaced_resource_quota(namespace=namespace, body=body)
     return quota
+
+
+def get_endpoint_ingress(name, namespace):
+    extensions_api_instance = client.ExtensionsV1beta1Api(config.load_kube_config())
+    api_response = extensions_api_instance.read_namespaced_ingress(name, namespace)
```

Comment:
Any handling in case of error?

Reply:
Hmm this is confusing, as in the beginning we agreed to an assumption that in tests function everything works fine so we are not going to catch errors (even [here](https://github.com/IntelAI/inference-model-manager/blob/master/tests/conftest.py#L72)) and after that a few calls appeared which has an error handling. So it is a matter of what path we want to go